frontend/src/static/css/style.scss

```diff
@@ -222,6 +222,12 @@ header {
   text-align: center;
 }
 
+.button {
```

Comment:
This global variable for `button` sets all existing buttons to have the same css property. Perhaps you could narrow the scope down to be within summary?

Reply:
There is no possibility to put it within summary. It's `summary-chart` and `summary-charts` as they both handle the different places where we have to use this button css. And if we do it, we will move back to what I did initially in this PR. 

Also, write now we use the `button` only within the summary and no place else. Also, I have written `button` CSS in such a way that all affected buttons are still the same (UI wise)

Reply:
Once we refactor out summary CSS then it will be possible to do what you are suggesting as button CSS will move with that. And to make sure that happens I have written `button` css inside `#summary`